dags/glam.py

```diff
@@ -135,6 +146,28 @@
     arguments=('--replace',),
     dag=dag)
 
+historgam_percentiles = bigquery_etl_query(
+    task_id="historgam_percentiles",
+    destination_table="historgam_percentiles_v1",
+    dataset_id=dataset_id,
+    project_id="moz-fx-data-shared-prod",
+    owner="msamuel@mozilla.com",
+    email=["telemetry-alerts@mozilla.com", "msamuel@mozilla.com"],
+    date_partition_parameter=None,
+    arguments=('--replace',),
+    dag=dag)
+
+user_counts = bigquery_etl_query(
+    task_id="user_counts",
+    destination_table="user_counts",
```

Comment:
I think we should make this name clearer, and version it: `glam_user_counts_v1`?